diagnostic_rt_qpcr/operation_types/run_qpcr/documentation.rb

```diff
@@ -1,23 +1,5 @@
-  # Default parameters that are applied equally to all operations.
-  #   Can be overridden by:
-  #   * Associating a JSON-formatted list of key, value pairs to the `Plan`.
-  #   * Adding a JSON-formatted list of key, value pairs to an `Operation`
-  #     input of type JSON and named `Options`.
-  #
-  def default_job_params
     {
+      thermocycler_model: TestThermocycler::MODEL,
```

Comment:
This is going to make it so that all the operations in the job have to use the same type of thermocycler. Is that desirable?

Reply:
I swear I moved that to Default operation_parameters...    Regardless that is not desirable at all

Reply:
Ahhh i fixed it in the protocol but not in the documentation